src/components/victory-voronoi.jsx

```diff
@@ -71,26 +71,28 @@ class VictoryVoronoi extends React.Component {
 
       const path = "M" + vertices.join("L") + "Z";
 
-      return (
+      const _buildSvg = voronoi => {
```

Comment:
Good call--that would be way more readable--but it looks `victory-animation` requires a single `data` prop that's passed on to the functional child (here, `_buildSvg`), so I think we're stuck with using just one argument. :/